sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -351,6 +351,10 @@ task chicagoTaxiExample {
 
 task validatesContainer() {
   def pyversion = "${project.ext.pythonVersion.replace('.', '')}"
+  if (project.hasProperty("pipExtraOptions")) {
```

Comment:
this is confusing when you read it outside of the context of this change.

Reply:
pipExtraOptions in this context is a very generic flag and its unclear how it ties to regenerating requirements.

Reply:
in the context  `validatesContainer()` task it makes sense to use a more specific flag name (for example `regenerateRequiremetns`), while in the context of Dockerfile (where we run pip) it does make sense to use a generic flag name (`pipExtraOptions`).

Reply:
Sg. i will refactor the names